gulpfile.js

```diff
@@ -115,14 +117,12 @@ gulp.task('js', ['js:combine'], function () {
     .pipe(uglify({
       mangle: false,
       preserveComments: false,
-      output: {
-        beautify: true
```

Comment:
Removing this setting lets the JS fully minify. Just pointing out the change I made.